.ci/pipeline/release_jjb.yaml

```diff
@@ -0,0 +1,65 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    folder: libxlio
+    properties:
+        - github:
+             url: "https://github.com/Mellanox/libxlio"
+        - build-discarder:
+            days-to-keep: 120
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: false
+    parameters:
+        - string:
+            name: "release_tag"
+            default: ""
+            description: "Tag to release"
+        - validating-string:
```

Comment:
This is nice but not working on Blue UI

![image](https://github.com/user-attachments/assets/c0400f8a-dcf0-4cb1-b111-1703fd0f4ee4)